pkg/plugins/shell/main_test.go

```diff
@@ -75,3 +75,35 @@ func TestShell_New(t *testing.T) {
 		})
 	}
 }
+
+func TestShell_customCommand(t *testing.T) {
```

Comment:
Testing internal methods is not necessary, this behavior is already ensured by your public methods tests.